fury/gltf.py

```diff
@@ -32,7 +33,8 @@
 
 
 class glTF:
-    def __init__(self, filename, apply_normals=False):
+    @warn_on_args_to_kwargs()
```

Comment:
parenthesis not needed everywhere

Reply:
if I don't put a parenthesis it raises a `TypeError`.